src/backend/distributed/master/master_node_protocol.c

```diff
@@ -666,6 +666,14 @@ GetTableCreationCommands(Oid relationId, bool includeSequenceDefaults)
 		tableDDLEventList = lappend(tableDDLEventList, tableColumnOptionsDef);
 	}
 
+	{
+		char *tableOwnerDef = pg_get_tableowner_string(relationId);
+		if (tableOwnerDef != NULL)
+		{
+			tableDDLEventList = lappend(tableDDLEventList, tableOwnerDef);
```

Comment:
We should probably verify whether a user can always make itself owner:

> To alter the owner, you must also be a direct or indirect member of the new owning role, and that role must have CREATE privilege on the table's schema. (These restrictions enforce that altering the owner doesn't do anything you couldn't do by dropping and recreating the table. However, a superuser can alter ownership of any table anyway.) 

It seems that I can do `ALTER TABLE me TO me` in PG11 even if I don't have create privileges on the schema, but we should have a test that makes sure this is also true for 9.6 and 10 and remains true in future versions.